core/tests/protractor/editorAndPlayer.js

```diff
@@ -205,6 +205,7 @@ describe('Full exploration editor', function() {
       editor.setTitle('Testing multiple rules');
       editor.setCategory('Answer Groups');
       editor.setObjective('To assess happiness.');
+      editor.previewSummaryTile();
```

Comment:
Seems like you want to separate this into editor.openSummaryTilePreview() and editor.closeSummaryTilePreview(), since they are represented by different user actions.